### PR TITLE
Polish CaseStatusSelectGroup dialog when changing status

### PR DESCRIPTION
**Pull request by wssheldon**
Screenshot 2023-12-18 at 11 45 27 AM
Screenshot 2023-12-18 at 11 45 20 AM

### CODE REVIEW

File: src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
+      </v-card>
+    </v-dialog>
+
+    <v-dialog v-model="alreadySelectedDialog" max-width="600">
+      <v-card class="mx-auto">
+        <v-card-title class="ml-2">Status Not Changed</v-card-title>
+        <v-card-text>
+          This case was moved to the status
+          <v-chip :color="statusColors[selectedStatus]">{{ selectedStatus }}</v-chip> on
+          {{ selectedStatusTooltip }}
```
Comment (whitdog47) [inline src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue:26]: Do you want to format this better and maybe add in the alt time zone tooltip?

Comment (wssheldon) [reply src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue:26]: By formatting do you mean the code formatting or the format of the text on the page? The former is done for me automatically by the linter.

Comment (whitdog47) [reply src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue:26]: Sorry I left off the important bit: format the time like with `formatToUTC`

Comment (wssheldon) [reply src/dispatch/static/dispatch/src/case/CaseStatusSelectGroup.vue]: Rgr, done.

**Pull request merged**